test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -283,6 +283,25 @@ def _init_test_roi_heads_faster_rcnn():
         box_score_thresh, box_nms_thresh, box_detections_per_img)
     return roi_heads
 
+def _construct_tensor_for_quantization_test(shape, offset=None, max_val=None):
+    """Due to difference in implementation details between PyTorch and ONNXRuntime, randomly generated
```

Comment:
Very much a nit, but: Prefer starting the doc with a single line summary: https://google.github.io/styleguide/pyguide.html#381-docstrings